lesson2/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>lesson2</title>
+</head>
+
+<body>
+    <script>
+        'use strict'
+
+        // 1
+        var a = 1,
+            b = 1,
+            c, d;
+        c = ++a;
+        alert(c); // значение увеличивается на 1 и инкремирование выполняется сразу, префиксная сумма = 2
+        d = b++;
+        alert(d); // значение увеличивается на 1 и инкремирование выполняется после, постфиксная сумма = 1
+
+        // В постфиксной форме сначала происходит возвращение значения, а потом выполняется инкремирование. В префиксной форме инкремирование  производится сразу, а возврат — уже с обновленным значением.
+        c = (2 + ++a);
+        alert(
+            c
+        ); // a уже равно 2 и мы опять увеличиваем ее на единицу и инкремирование выполняется сразу, соответственно 2 + 3 = 5
+        d = (2 + b++);
+        alert(d); // b уже равно 2, мы используем постфиксную сумму и выполняем сложение 2 + 2 = 4
+        alert(a); // изначально значение а = 1, после выполнили увеличение на 1 два раза с помощью префиксной сумма = 3
+        alert(
+            b
+        ); // изначально значение b = 1, после увеличения на 1 два раза b = 3 выполнили с помощью постфиксной суммы, значение присваивается в следующей операции
+
+        // 2
+
+        var a = 2;
+        var x = 1 + (a *= 2);
+        // х будет равен 5, значение а, равное 2, увеличится в 2 раза, с помощью оператора умножения а *= 2 эквивалентно а = а * 2
+
+        // 3
+
+        function sum(a, b) {
+            if (a >= 0 && b >= 0) {
+                console.log(a - b);
+            } else if (a < 0 && b < 0) {
+                console.log(a * b);
+            } else if (a >= 0 && b < 0 || a < 0 && b > 0) {
```

Comment:
Здесь не нужно дополнительное условие, т.к. нас интересуют "все остальные случаи", которые как раз покрываются обычным else
```suggestion
            } else {
```